main.go

```diff
@@ -121,6 +122,7 @@ func addHealthAndReadyChecks(mgr manager.Manager) {
 // getRemoteClusterClientConfig reads a config file with a remote cluster configuration and returns
 // the pointer to a rest.Config object.
 func getRemoteClusterClientConfig(configFile string) (*rest.Config, error) {
+	/* #nosec G304 */
```

Comment:
what's this for?

Reply:
Probably gosec complains.

Reply:
To disable the security check that was causing the CI to fail. We need to consume the config from a file, so it's needed.